code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -780,45 +780,13 @@
 	var/recharge_delay = 200 //How long after we've been shot before we can start recharging. 20 seconds here
 	var/recharge_cooldown = 0 //Time since we've last been shot
 	var/recharge_rate = 1 //How quickly the shield recharges once it starts charging
```

Comment:
are any of these vars on the hardsuits (current_changes, max_charges, recharge stuff) even used anymore?

Reply:
oh hell good point

Reply:
nevermind apparently those are used, implementing them instead

Reply:
you might need to do some snowflake for the arcane charge wizards get for their shielded hardsuit